CHANGELOG.md

```diff
@@ -4,14 +4,145 @@ All notable changes to this project will be documented in this file.
+
+    Install type | Old path | New path
+    ------------ | -------- | ---------
+    Composer     | `vendor/wimg` | `vendor/wimg/php-compatibility`
+    Unzipped release to arbitrary directory | `path/to/dir/abovePHPCompatibility` | `path/to/dir/abovePHPCompatibility/PHPCompatibility`
+    Git checkout | `path/to/dir/abovePHPCompatibility` | `path/to/dir/abovePHPCompatibility/PHPCompatibility`
+    PEAR         | If the old install instruction has been followed, not registered. | `path/to/PHPCompatibility`
+    
+    > **Side-note**:
+	>
+	> If you used the old install instructions for a PEAR install, i.e. checking out the latest release to the `PHP/CodeSniffer/Standards/PHPCompatibility` directory, and you intend to upgrade to PHP CodeSniffer 3.x, it is recommended you move the PHPCompatibility folder out of the PEAR directory now, as the layout of the PHPCS directory has changed with PHPCS 3.x and you may otherwise lose your PHPCompatibility install when you upgrade PHP CodeSniffer via PEAR.
+
+3. There are two ways in which you can register the new `installed_paths` value with PHP CodeSniffer. Choose your preferred method:
+    * Run `phpcs --config-set installed_paths ...` and include all previously installed paths including the _adjusted_ path for the PHPCompatibility standard.
+
+        For example, if the previous value of `installed_paths` was
+		
+		`/path/to/MyStandard,/path/to/dir/abovePHPCompatibility`
+
+		you should now set it using
+
+		`phpcs --config-set installed_paths /path/to/MyStandard,/path/to/PHPCompatibility`
+
+    * If you use a custom ruleset in combination with PHPCS 2.6.0 or higher, you can pass the value to PHPCS from your custom ruleset:
+        ```xml
+        <config name="installed_paths" value="vendor/wimg/php-compatibility" />
+        ```
+4. Run `phpcs -i` to verify that the PHPCompatibility standard is now listed again in the list of installed standards.
+
+
+### Upgrading to PHPCS 3.x
+
+The path to the `phpcs` script has changed in PHPCS 3.x which will impact how you call PHPCS.
+
+Version | PHPCS 2.x | PHPCS 3.x
+------- | --------- | ---------
+Generic `phpcs` Command | `path/to/PHP_CodeSniffer/scripts/phpcs ....` | `path/to/PHP_CodeSniffer/bin/phpcs ....`
+Composer command | `vendor/bin/phpcs ...` | `vendor/bin/phpcs ...`
+
+So, for Composer users, nothing changes. For everyone else, you may want to add the `path/to/PHP_CodeSniffer/bin/phpcs` path to your PATH environment variable or adjust any scripts - like build scripts - which call PHPCS.
+
+
+### Upgrading a Travis build script
+
+If you run PHPCompatibility against your code as part of your Travis build:
+* If you use Composer to install PHP CodeSniffer and PHPCompatibility on the travis image and you've made the above mentioned changes to your `composer.json` file, your build should pass again.
```

Comment:
Nit: I can't find any "above mentioned changes to composer.json". 
You probably just meant the adjustments of the version numbers?